config/crd/bases/infrastructure.cluster.x-k8s.io_nutanixclustertemplates.yaml

```diff
@@ -61,20 +66,19 @@ spec:
                         - port
                         type: object
                       failureDomains:
-                        description: failureDomains configures failure domains information
-                          for the Nutanix platform. When set, the failure domains
-                          defined here may be used to spread Machines across prism
-                          element clusters to improve fault tolerance of the cluster.
+                        description: |-
```

Comment:
are these changes required?

Reply:
These seem to be autogenerated. My guess is @yanhua121 is not using devbox for her dependencies and as such the versions of dependencies are different.

@yanhua121 please install devbox and direnv as suggested in the pre-requisites section under https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/blob/main/docs/developer_workflow.md#prerequisites